Classes/Manager/Submanagers/OCTSubmanagerAvatars.m

```diff
@@ -38,11 +43,12 @@ - (void)setAvatar:(UIImage *)avatar
         [fileManager createDirectoryAtPath:[path stringByDeletingLastPathComponent]
                withIntermediateDirectories:YES
                                 attributes:nil
-                                     error:nil];
-        [data writeToFile:path atomically:NO];
+                                     error:error];
+        success = [data writeToFile:path atomically:NO];
```

Comment:
It would be even better to use `writeToFile:options:error:` here so that we can pass error to user of method.